operator-common/src/main/java/io/strimzi/operator/common/DefaultAdminClientProvider.java

```diff
@@ -22,10 +22,10 @@
 
     @Override
     public Admin createAdminClient(String hostname, Secret clusterCaCertSecret, Secret keyCertSecret, String keyCertName) {
-
+        PasswordGenerator pg = new PasswordGenerator(12);
```

Comment:
Could this line be next to the line where the password is generated? It will be a bit more readable.